src/azure/cli/_config.py

```diff
@@ -0,0 +1,55 @@
+#---------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+#---------------------------------------------------------------------------------------------
+import os
+from six.moves import configparser
+
+GLOBAL_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'az_config'))
+ACTIVE_ENV_CONFIG_PATH = os.path.expanduser(os.path.join('~', '.azure', 'env_config', 'default'))
+ENV_VAR_PREFIX = 'AZURE_CLI_'
+
+_UNSET = object()
+_ENV_VAR_FORMAT = ENV_VAR_PREFIX+'{section}_{option}'
+
+class AzConfig(object):
+    _BOOLEAN_STATES = {'1': True, 'yes': True, 'true': True, 'on': True,
```

Comment:
if the user has to type these values, I vote for also supporting y and n.  If this is just config file storage, I'm fine with it how it is.

---
In reply to: [76105114](https://github.com/Azure/azure-cli/pull/739#discussion_r76105114) [](ancestors = 76105114)